src/index.ts

```diff
@@ -381,5 +385,7 @@ async function handleProgramArgs() {
 
 void handleProgramArgs().catch(error => {
   console.error(error);
+  const { args } = getProgramArgs();
+  console.error(`${args.operand} process failed`);
```

Comment:
This causes the message `undefined process failed` to be printed when the API exits from an error when started regularly e.g. `node ./lib/index.js`

Reply:
In that case, it won't go for events and [initiate the application](https://github.com/hirosystems/stacks-blockchain-api/blob/master/src/index.ts#L331) instead. That has its own [error handler](https://github.com/hirosystems/stacks-blockchain-api/blob/master/src/index.ts#L219-L228) which exits the program. 
Please let me know if I'm wrong here.

Reply:
Possibly, there's async and sync code mixed in here. Better to just check for null/undefined and avoid logging